drivers/bridge/bridge.go

```diff
@@ -562,6 +562,11 @@ func (d *driver) CreateEndpoint(nid, eid types.UUID, epInfo driverapi.EndpointIn
 	}
 	endpoint.macAddress = mac
 
+	// Up the host interface after finishing all netlink configuration
+	if err := netlink.LinkSetUp(host); err != nil {
```

Comment:
Given all the challenges we faced with Centos6.6, can you please confirm if this call works fine in that older kernel ?